src/course-outline/card-header/CardHeader.tsx

```diff
@@ -267,14 +269,14 @@ const CardHeader = ({
                   <Dropdown.Item
                     data-testid={`${namePrefix}-card-header__menu-move-up-button`}
                     onClick={onClickMoveUp}
-                    disabled={!actions.allowMoveUp}
+                    disabled={!actions.allowMoveUp || parentInfo?.hasUpstream}
                   >
                     {intl.formatMessage(messages.menuMoveUp)}
                   </Dropdown.Item>
                   <Dropdown.Item
                     data-testid={`${namePrefix}-card-header__menu-move-down-button`}
                     onClick={onClickMoveDown}
-                    disabled={!actions.allowMoveDown}
+                    disabled={!actions.allowMoveDown || parentInfo?.hasUpstream}
```

Comment:
It does not seem trivial, but have you tried changing the `action.allowMoveDown` to account for the parent upstream information?